website/netlify.toml

```diff
@@ -67,7 +67,10 @@
   to = "/docs/add-secure-apps/flows-stages/flow/executors/if-flow.md"
   status = 302
 
-
+[[redirects]]
+  from = "/docs/customize/brands.md"
+  to = "/docs/customize/branding.md"
```

Comment:
```suggestion
  from = "/docs/customize/brands"
  to = "/docs/customize/branding"
```

Reply:
thanks, @BeryJu I fixed that and also the two lines above lol